gcloud-java-dns/src/main/java/com/google/gcloud/dns/ChangeRequest.java

```diff
@@ -189,7 +189,7 @@ static ChangeRequest fromPb(Dns dns, String zoneName, Change pb) {
   static Function<Change, ChangeRequest> fromPbFunction(final Dns dns, final String zoneName) {
     return new Function<Change, ChangeRequest>() {
       @Override
-      public ChangeRequest apply(com.google.api.services.dns.model.Change pb) {
```

Comment:
This is a hard one. /cc @mziccard @ajkannan 

So, the warning has it merits (I would imagine in most cases) however we tried to keep the underlying model/pb classes using a fully-qualified package notation to make the distinction clear.

Maybe it is not worth doing it anymore but probably worth some discussion/consensus.
Also, seems like there are plenty of other cases in this module where this change was not made
including in Zone, ZoneInfo, ProjectInfo, DnsImpl and some tests.